contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -3786,6 +3787,11 @@ sp_rename_internal(PG_FUNCTION_ARGS)
 			objtype_code = OBJECT_TABLE;
 			process_util_querystr = "(ALTER TABLE )";
 		}
+		else if (strcmp(objtype, "IX") == 0)
+		{
+			objtype_code = OBJECT_INDEX;
+			process_util_querystr = "(ALTER INDEX )";
```

Comment:
Macro might be better

Reply:
We are already having such hardcoded command names for all other kind of objects in this function. Moreover, macro is mostly useful when it is being used at multiple places, which is not true for this case so I think we can keep it hardcoded for now.